application/actions/post_test.go

```diff
@@ -289,3 +293,21 @@ func (as *ActionSuite) Test_UpdatePostStatus() {
 		}
 	}
 }
+
+func (as *ActionSuite) Test_SearchRequests() {
+	f := createFixturesForSearchRequestsQuery(as)
+	query := `{ searchRequests(text: "match")
```

Comment:
If you change this, you can use `PostsResponse` rather than `SearchRequestsResponse`
```suggestion
	query := `{ posts: searchRequests(text: "match")
```

Reply:
Good catch.